cvat-ui/package.json

```diff
@@ -76,7 +76,10 @@
     "react-color": "^2.19.3",
     "react-cookie": "^4.0.3",
     "react-dom": "^16.14.0",
+    "react-hotkeys": "^2.0.0",
```

Comment:
Please, merge current develop before. We do not use "react-hotkeys" anymore.

Reply:
Done